lib/improver/threshold.py

```diff
@@ -138,12 +138,6 @@ def process(self, input_cube):
                 truth_value = 1. - truth_value
             cube.data = truth_value
 
-            # TODO: Correct when formal cf-standards exists
-            # Force the metadata to temporary conventions
-            if self.below_thresh_ok:
-                cube.attributes.update({'relative_to_threshold': 'below'})
-            else:
-                cube.attributes.update({'relative_to_threshold': 'above'})
             cube.rename("probability_of_{}".format(cube.name()))
```

Comment:
The `rename` can also be moved out of the loop.
Please also copy the `# TODO` statement to this point so that the threshold dimension is revisited at some point.

Reply:
And the the changing the cube units can be moved as well as you're at it.